boot/espressif/main.c

```diff
@@ -103,6 +136,36 @@ int main()
     }
 #endif
 
+#ifdef CONFIG_SECURE_FLASH_ENC_ENABLED
+    /* Step 4, 5 & 6 (see above for full description):
+     *   4) Generate Flash Encryption key and write to EFUSE.
+     *   5) Encrypt flash in-place including bootloader, image primary/secondary slot and scratch.
+     *   6) Burn EFUSE to enable flash encryption
+     *      (FLASH_CRYPT_CNT for ESP32 / SPI_BOOT_CRYPT_CNT for ESP32S2, ESP32C3)
```

Comment:
```suggestion
     *   6) Burn EFUSE to enable flash encryption
```
We may omit the eFuses identifiers here.

Reply:
Done